hex-literal/src/lib.rs

```diff
@@ -110,7 +119,7 @@ impl TokenTreeIter {
                 b'A'..=b'F' => v - 55,
                 b'a'..=b'f' => v - 87,
                 b' ' | b'\r' | b'\n' | b'\t' => continue,
-                _ => panic!("encountered invalid character"),
+                c => panic!("encountered invalid character: `{}`", c as char),
```

Comment:
Note that it will print a wrong error message for non-ASCII characters. It's probably worth to change this line into something like this:
```rust
0..=127 => panic!("encountered invalid character: `{}`", c as char),
_ => panic!("encountered invalid non-ASCII character"),
```

Reply:
Good catch--thanks! Updated.